src/main/java/core/basesyntax/service/impl/ShopServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitDao;
```

Comment:
Import statements should not be separated by empty lines. Please remove the redundant empty line.

Reply:
++